onebox/start_onebox.sh

```diff
@@ -89,30 +89,6 @@ test -d recycle_bin3 && rm -rf recycle_bin3
                    --zk_cluster=${ZK_CLUSTER}\
                    --zk_root_path=/onebox > ns3.log 2>&1 &
 
-# start TaskManager if exists
```

Comment:
why delete? Don't we need to start task manager when starting onebox?

Reply:
The `start_onebox.sh` is executed by for `mvn package`. The java target path does not exist when starting onebox. Now we change to start TaskManager in java unit test file which may use the latest code of TaskManager.

Reply:
But the normal user can't start task manager by `start_onebox.sh`, it's not good.

Reply:
Normal users may use `start-all.sh` in release to setup the all-in-one cluster. IMO, `onebox` is used for unit tests currently.

Reply:
I see, so `start_onebox.sh` is only for ut(I've forgot start-all.sh is for cluster, it's for standalone before). It's fine, but this script should be renamed.